packages/wallets/src/features/cashier/WalletCashier.scss

```diff
@@ -1,5 +1,5 @@
 .wallets-cashier-content {
-    padding: 24px;
+    padding: 24px 24px 0px;
```

Comment:
@aum-deriv please change it to `rem` due to this PR
https://github.com/binary-com/deriv-app/pull/10881/